params/index.d.ts

```diff
@@ -24,6 +30,12 @@ interface Params {
    * @param input Template string.
    * @return Transform for translation.
    */
+  <Input extends string>(input: Input): TranslationFunction<
```

Comment:
In Router we were forced to define argument as `const` to be sure that TS will use `"/posts/:id"` type instead of just `string` https://github.com/nanostores/router/blob/main/index.d.ts#L132C30-L132C35

Why we don’t need it here?

Reply:
Bcs in router types it's needed to narrow types for the other consumers, that will be working with returned type. Here I don't see cases why it might be necessary, but I might be wrong)

Reply:
I was passing by and wanted to give a hint that using canonical URLs will be safer in the future. Just press `y` and the URL will become this: https://github.com/nanostores/router/blob/e927e23d509d1cde4140653071743a3c370de70b/index.d.ts#L132C30-L132C35